bower.json

```diff
@@ -28,7 +28,7 @@
     "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
     "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0"
+    "polymer": "Polymer/polymer#^1.1.0"
```

Comment:
Move the `<style>` tag inside the `<template>` tag is only from 1.1. :blush:

[1.1 Release](https://blog.polymer-project.org/announcements/2015/08/13/1.1-release/):
*Previously, we recommended that a `<style>` element should be placed inside an element’s `<dom-module>` but outside of its template. This is still supported, but we’ve now optimized placing styles within the template itself, so having the `<style>` tag outside of the template will be slower. We will be updating all of the elements provided by the Polymer team to match this new recommended behavior as well.*